x-pack/plugins/security_solution/public/management/components/console/components/command_list.tsx

```diff
@@ -289,21 +352,24 @@ export const CommandList = memo<CommandListProps>(({ commands, display = 'defaul
     );
 
     return (
-      <>
+      <div data-test-subj={getTestId()}>
         {commandsByGroups.map((commandsByGroup, i) => (
           <StyledEuiBasicTable
+            data-test-subj={getTestId(
+              convertToTestId(commandsByGroup[0].helpGroupLabel ?? otherCommandsGroupLabel)
+            )}
             key={`styledEuiBasicTable-${i}`}
             items={getTableItems(commandsByGroup)}
             columns={getTableColumns(commandsByGroup)}
           />
         ))}
         {callout}
-      </>
+      </div>
     );
   }
 
   return (
-    <>
+    <div data-test-subj={getTestId()}>
```

Comment:
React.Fragment also accepts data-test-subj if this is why you use div here

Reply:
I did not realize `React.Fragment` accepted a `data-data-subj`. that seems weird since a fragment does not really render anytime of DOM. Are you sure this is supported?

Reply:
Hmm I was sure when I was writting this, let me double check ;)

Reply:
It seems I got confused for a while, and it is about passing `key` prop to fragment. It doesnt support data-test-id, sorry about that 👍